app/views/metrics/index.html.erb

```diff
@@ -46,17 +46,17 @@
         <button type="submit" class="a-apply-button">Apply</button>
       </div>
 
-      <div class="m-search-filter">
-        <label for="search-departments">Find department</label>
-        <input type="text" id="search-departments" class="a-search-input" placeholder="Find department">
-        <button class="a-search-button">Search</button>
+      <div class="m-search-filter hidden" id="search-filter">
+        <label for="search-departments" class="visuallyhidden">Find department</label>
+        <input type="search" id="search-departments" class="a-search-input" placeholder="Find department">
+        <input class="a-search-button" type="button" value="Search">
       </div>
     <% end %>
   </div>
 
   <!--end of filter-->
 
-  <div class="metrics-groups grid-row">
+  <div class="metrics-groups grid-row" id="search-results">
```

Comment:
I've been using the `data-behaviour` attribute to layer on Javascript behaviour. My rationale for that is that it makes it clearer in the HTML which attributes are used for styling, and which are used for behaviour. However this isn't the one-true approach, so let's have a chat about this.